src/schema/objects/columns.yaml

```diff
@@ -52,6 +52,17 @@ color:
     Hexadecimal. Label color for visualization.
   type: string
   unit: hexadecimal
+compcor:
+  name: ^[a|t|w|c]_compcor_cor_[0-9a-zA-Z]+$
+  display_name: CompCor
+  description: |
+    CompCor (Behzadi, et al., 2007) calculated with voxels chosen based on:
+    a: anatomically derived ROIs (white matter and CSF),
+    t: temporal variance, w: white matter voxels only, c: CSF voxels only
+cosine_x:
+  name: ^cosine_[0-9a-zA-Z]+$
```

Comment:
These new regex-based names are new. I think having regex-based names fits, but one other novel aspect is that the columns will have a one-to-many relationship. All other columns and metadata fields have had one-to-one mappings so far (one object definition for each column/field). Do we need an additional field in the object schema to account for this?